tools/vera++/scripts/rules/jerry_dereference_operator_always_on_right.tcl

```diff
@@ -18,7 +18,9 @@
 foreach f [getSourceFileNames] {
     set lineNumber 1
     foreach line [getAllLines $f] {
-        if {[regexp {\w\*\s\w+} $line]} {
+        if {[regexp {\w\*\s\w+} $line]
+             || [regexp {\w\*\)} $line]
+             || [regexp {\w\*$} $line]} {
```

Comment:
I've made a terrible mistake. I've taken a look at the C99 standard :). Now I know that `*` is a punctuator, which is known as an operator, if (and only if) used in a context where it specifies an operation (6.4.6 p2). However, the first and the third checks are for declarators (6.7.5), while the second check is for the type-name part of a cast operator (6.5.4), which leads back again to declarations (6.7.6 p2).

Could jerry_pointer_declarator_space.tcl work as a filename, and "there should be a space between the referenced type and the pointer declarator" as a message work?